api/v1/controllers/samples.js

```diff
@@ -57,21 +93,33 @@ module.exports = {
    * @param {Function} next - The next middleware function in the stack
    */
   findSamples(req, res, next) {
+    // Check if Cache is on for Wildcard Sample query
+    if (featureToggles.isFeatureEnabled('cacheGetSamplesWildcard')) {
+      const query = req.query.name;
```

Comment:
if think if you do something like `const query = req.query.name || ''`; you can avoid the nested ternary in line 100.

Reply:
or changing it to 
`helper.cacheEnabled = query && (query.indexOf('*') > -1)` , will give the same effect. Correct?

Reply:
Good point. Correct. Thanks :)